olca-app-html/.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /dist/
 /images/background_start_page.jpg
+cache
```

Comment:
Nit picking: use `/cache/` as it is a folder.

Reply:
Thanks @francoislerall :)